docker-compose.test.yml

```
@@ -0,0 +1,42 @@
+
+  playwright:
+    build:
+      context: .
+      dockerfile: Playwright.Dockerfile
+    restart: unless-stopped
+    volumes:
+      - .:/app
+    ports:
+      - '9324:9324'
+    environment:
+         - NEXT_PUBLIC_PRODUCTION_API_URL=http://prod:3001
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 9

(indentation)